testfiles/Fault_ThermostatOffset_RefBldgMediumOfficeNew2004_Chicago.idf

```diff
@@ -7913,106 +7913,106 @@
-    Perimeter_mid_ZN_2 Thermostat; 		 !- Thermostat Name
+    1.0;                     !- Reference Thermostat Offset
 	
   FaultModel:ThermostatOffset,
     Ther_Offset_Zn23,		 !- Name
+    Perimeter_mid_ZN_3 Thermostat, 		 !- Thermostat Name
     ,                        !- Availability Schedule Name
     ,                        !- Severity Schedule Name
-    1.0,       				 !- Reference Thermostat Offset
-    Perimeter_mid_ZN_3 Thermostat; 		 !- Thermostat Name
+    1.0;                     !- Reference Thermostat Offset
 	
   FaultModel:ThermostatOffset,
     Ther_Offset_Zn24,		 !- Name
+    Perimeter_mid_ZN_4 Thermostat, 		 !- Thermostat Name
     ,                        !- Availability Schedule Name
     ,                        !- Severity Schedule Name
-    1.0,       				 !- Reference Thermostat Offset
-    Perimeter_mid_ZN_4 Thermostat; 		 !- Thermostat Name
+    1.0;                     !- Reference Thermostat Offset
 	
   FaultModel:ThermostatOffset,
     Ther_Offset_Zn30,		 !- Name
+    Core_top Thermostat, 		 !- Thermostat Name
     ,                        !- Availability Schedule Name
     ,                        !- Severity Schedule Name
-    1.0,       				 !- Reference Thermostat Offset
-    Core_top Thermostat; 		 !- Thermostat Name
+    1.0;                     !- Reference Thermostat Offset
 	
   FaultModel:ThermostatOffset,
     Ther_Offset_Zn31,		 !- Name
+    Perimeter_top_ZN_1 Thermostat, 		 !- Thermostat Name
     ,                        !- Availability Schedule Name
     ,                        !- Severity Schedule Name
-    1.0,       				 !- Reference Thermostat Offset
-    Perimeter_top_ZN_1 Thermostat; 		 !- Thermostat Name
+    1.0;                     !- Reference Thermostat Offset
 	
   FaultModel:ThermostatOffset,
     Ther_Offset_Zn32,		 !- Name
+    Perimeter_top_ZN_2 Thermostat, 		 !- Thermostat Name
     ,                        !- Availability Schedule Name
     ,                        !- Severity Schedule Name
-    1.0,       				 !- Reference Thermostat Offset
-    Perimeter_top_ZN_2 Thermostat; 		 !- Thermostat Name
+    1.0;                     !- Reference Thermostat Offset
 	
   FaultModel:ThermostatOffset,
     Ther_Offset_Zn33,		 !- Name
+    Perimeter_top_ZN_3 Thermostat, 		 !- Thermostat Name
     ,                        !- Availability Schedule Name
     ,                        !- Severity Schedule Name
-    1.0,       				 !- Reference Thermostat Offset
-    Perimeter_top_ZN_3 Thermostat; 		 !- Thermostat Name
+    1.0;                     !- Reference Thermostat Offset
 	
   FaultModel:ThermostatOffset,
     Ther_Offset_Zn34,		 !- Name
+    Perimeter_top_ZN_4 Thermostat, 		 !- Thermostat Name
     ,                        !- Availability Schedule Name
     ,                        !- Severity Schedule Name
-    1.0,       				 !- Reference Thermostat Offset
-    Perimeter_top_ZN_4 Thermostat; 		 !- Thermostat Name
+    1.0;                     !- Reference Thermostat Offset
 	
```

Comment:
@mjwitte I see what you mean. I've no idea what it is. 
![capture](https://cloud.githubusercontent.com/assets/7785946/6693383/e42fb3f0-cc8f-11e4-99a7-9071ff04c080.JPG)

I tried to clean this up by copying the several ending lines from other IDF files to this one, but it did not work. Hopefully it doesn't matter. @Myoldmopar ?